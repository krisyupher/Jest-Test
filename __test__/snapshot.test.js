import {getCharacter} from '../snapshot';
import rick from "../rick.json";

describe("Es hora de las instantaneas", ()=>{
	test("snapshot", () =>{
		expect(getCharacter(rick)).toMatchSnapshot();
	})
	test('Instant치nea con algunas Excepciones en sus paths', () => {
    const user = {
      id: Math.floor(Math.random() * 100),
      name: 'Alejandro Gonz치lez Reyes',
      profesion: 'JavaScript and PHP Development',
      createdAt: new Date
    }

    // Se espera que la data recibida coincida con la instant치nea. A excepci칩n de los campos id y createdAt, los cuales se espera que reciban cualquier Numero y Fecha respectivamente
    expect(user).toMatchSnapshot({
      id: expect.any(Number),
      createdAt: expect.any(Date)
    })
  })
})