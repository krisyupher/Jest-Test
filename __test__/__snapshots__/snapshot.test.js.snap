// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las instantaneas Instantánea con algunas Excepciones en sus paths 1`] = `
Object {
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "Alejandro González Reyes",
  "profesion": "JavaScript and PHP Development",
}
`;

exports[`Es hora de las instantaneas snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;
